### PR TITLE
fix/wasm-vm: error in serialization/deserialzation

### PR DESCRIPTION
**Pull request by Pilou97**
# Problem:
Impossible to apply a vm operation

# Solution:
Fix the serialization and deserializarion thanks to @Zett98

### CODE REVIEW

**Review by Zett98 — CHANGES_REQUESTED** [review]
This line should be dropped

File: deku-p/src/core/protocol/protocol.ml [hunk 1]
```diff
@@ -110,6 +112,7 @@ let apply_operation ~current_level protocol operation :
               with
               | vm_state, ledger -> Some (ledger, Some receipt, vm_state, None)
               | exception External_vm_client.Vm_execution_error error ->
+                  let () = failwith error in
```
Comment (Zett98) [inline deku-p/src/core/protocol/protocol.ml:115]: This is a mistake from my end, forgot to drop it before sending the patch

Comment (Pilou97) [reply deku-p/src/core/protocol/protocol.ml:115]: no worries, I'll remove it

**Review by gsebil08 — APPROVED** [review]
Tested on my side, was able to mint a cookie!!

**Review by d4hines — APPROVED** [review]
LGTM

**Pull request merged**